### PR TITLE
Error bar should not be overlayed on top of panel

### PR DESCRIPTION
**Pull request by carocao-msft**
# What
Error bar should not be overlayed on top of panel

# Why
https://skype.visualstudio.com/SPOOL/_workitems/edit/3128672

# How Tested
Tested with calling and call with chat samples
Screenshot 2023-03-27 at 10 30 50 PM

Screenshot 2023-03-27 at 10 30 55 PM

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-composites/tests/browser/snapshots/beta/tests/browser/call/hermetic/ErrorBar.test.ts-snapshots/failure-to-start-video-on-error-bar-Mobile-Android-Landscape-linux.png
[the remark this answers is not in the document]
Comment (carocao-msft) [reply packages/react-composites/tests/browser/snapshots/beta/tests/browser/call/hermetic/ErrorBar.test.ts-snapshots/failure-to-start-video-on-error-bar-Mobile-Android-Landscape-linux.png:1]: I don't see a difference too. This PR should not trigger any snapshot updates, I only ran it to remove the wrong snapshots from earlier build

[the remark this answers is not in the document]
Comment (dmceachernmsft) [reply packages/react-composites/tests/browser/snapshots/beta/tests/browser/call/hermetic/ErrorBar.test.ts-snapshots/failure-to-start-video-on-error-bar-Mobile-Android-Landscape-linux.png:1]: seems there is a minute colour difference for some reason in the icons
image

File: packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx [hunk 1]
```diff
@@ -188,11 +177,22 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
           <Stack horizontal grow>
             <Stack.Item grow style={callCompositeContainerFlex()}>
               <Stack.Item styles={callGalleryStyles} grow>
-                {props.onRenderGalleryContent && (
-                  <Stack verticalFill styles={mediaGalleryContainerStyles}>
-                    {props.onRenderGalleryContent()}
-                  </Stack>
-                )}
+                <Stack verticalFill styles={mediaGalleryContainerStyles}>
```
Comment (dmceachernmsft) [inline packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx:180]: How are these changes affecting the mobile experience? are we supposed to see the error bars when the mobile panes are open as well?

Comment (carocao-msft) [reply packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx:180]: Screenshot 2023-03-30 at 1 40 34 PM
Screenshot 2023-03-30 at 1 40 27 PM

**Review by dmceachernmsft — APPROVED** [review]

**Review by prabhjot-msft — APPROVED** [review]

**Pull request merged**